lstmcpipe/scripts/script_batch_filelist_rta.py

```diff
@@ -38,22 +38,26 @@ def main():
     parser.add_argument(
         "--keep_file",
         "-k",
-        action='store_true',
+        action="store_true",
         dest="keep_file",
         help="Keep output of hiperta. Set by default to False",
     )
     parser.add_argument(
         "--debug_mode",
         "-d",
-        action='store_true',
+        action="store_true",
         dest="debug_mode",
         help="Activate debug mode (add cleaned mask in the output hdf5). Set by default to False",
     )
     args = parser.parse_args()
 
-    task_id = int(os.environ["SLURM_ARRAY_TASK_ID"])
-    file_for_this_job = args.file_list[task_id]
-    print("Using file: ", file_for_this_job)
+    task_id = int(environ.get("SLURM_ARRAY_TASK_ID"))
```

Comment:
This won't help since `int(None)` will fail. You can do `task_id = int(environ.get("SLURM_ARRAY_TASK_ID", -1))` and then check for `== -1`